src/entries/popup/pages/swap/onSwap.tsx

```diff
@@ -91,10 +93,12 @@ export const onSwap = async ({
     tradeAmountUSD: q.tradeAmountUSD,
     crosschain: assetToSell.chainId !== assetToBuy.chainId,
     degenMode,
-    hardwareWallet: (await wallet).type === KeychainType.HardwareWalletKeychain,
+    hardwareWallet: {
+      isHardwareWallet:
+        (await wallet).type === KeychainType.HardwareWalletKeychain,
+      vendor: (await wallet).vendor,
+    },
```

Comment:
There is no nice way to migrate event keys once they're shippewd, so I'd just use `hardwareWallet` and `hardwareWalletVendor` here instead of the object